.github/scripts/test-project.sh

```diff
@@ -35,12 +35,17 @@ then
   true
   # if a project needs to be skipped for any reason, replace `foobar` with its folder name or add additional conditions
 else
+  default_version="$(cat .github/prisma-version.txt)"
+  cli_version_dev="$(node -e "console.log(require('./$dir/$project/package.json').devDependencies.prisma ?? '')")"
+  cli_version_dep="$(node -e "console.log(require('./$dir/$project/package.json').dependencies.prisma ?? '')")"
+  version="$(node -e "console.log('$cli_version_dev' || '$cli_version_dep' || '$default_version')")"
+
   schema_path=$(find $dir/$project -name "schema.prisma" ! -path "*/node_modules/*" | head -n 1)
   if grep -q "env(\"DATABASE_URL\")" "$schema_path"; then
     echo ""
     echo "found 'schema.prisma' with 'env(\"DATABASE_URL\")': $schema_path"
-    echo "npx prisma db push --accept-data-loss --skip-generate --schema=$schema_path"
-    npx prisma db push --accept-data-loss --skip-generate --schema=$schema_path
+    echo "pnpm dlx prisma@$version db push --accept-data-loss --skip-generate --schema=$schema_path"
+    pnpm dlx prisma@$version db push --accept-data-loss --skip-generate --schema=$schema_path
```

Comment:
Small addition so that the integration version is used in the project, instead of the globally installed `prisma`. This is better for consistency when working with with integration versions and ensures (best effort) that we are using the integration end-to-end, even in automatic db creation.

Reply:
This is actually needed, because `prisma@5.2.0` is not aware of the `driverAdapters` preview feature.